VirtoCommerce.InventoryModule.Web/Scripts/blades/inventory-fulfillmentcenters-list.js

```diff
@@ -10,16 +10,36 @@ angular.module('virtoCommerce.inventoryModule')
             }, 0, false);
     });
 
-    blade.refresh = function () {
+    blade.clearKeyword = function() {
+        blade.searchKeyword = undefined;
+        blade.refresh();
+    };
+
+    blade.refresh = function() {
         blade.isLoading = true;
-        return blade.parentWidgetRefresh().$promise.then(function (results) {
+        return blade.parentWidgetRefresh().$promise.then(function(results) {
             blade.isLoading = false;
             blade.currentEntities = results;
 
+            var grouped = _.groupBy(blade.currentEntities,
```

Comment:
Search will work only on one part of centers that are loaded by inventory widget.